src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -1108,5 +1120,198 @@ public void PopulateCommand(SqlCommand sqlCommand, short resourceTypeId, long st
                 _up.AddParameter(sqlCommand.Parameters, up);
             }
         }
+
+        private class ResourceSearchParamStats
+        {
+            private ISqlRetryService _sqlRetryService;
```

Comment:
Should this attribute be readonly? I only see it being initialized in the constructor.

Reply:
done